tests/plugins/pastefromword/manual/pasteimagessafari.html

```diff
@@ -0,0 +1,7 @@
+<textarea id="editor" cols="30" rows="10"></textarea>
+<script>
+	if ( !CKEDITOR.env.safari ) {
+		bender.ignore();
+	}
```

Comment:
Maybe it makes sense to keep this test also for different browsers to make sure we didn't introduce regression?

Reply:
All other browsers are covered by unit tests for Paste Tools, Paste from Word and Paste from Libre Office, so I don't think it would be necessary.